CSS updates

skoolbus39 opened the pull request:
Update book pages block menu styling:
- Updated padding, font-size, colors, background-colors
- Use Unicode glyphs instead of raster images

skoolbus39 committed:
Remove raster images from list items

skoolbus39 committed:
Update book pages block menu styling

* Updated padding, font-size, colors, background-colors
* Use Unicode glyphs instead of raster images

ericras commented on css/style.css:
```diff
@@ -208,3 +208,9 @@ input.webform-calendar {
 }
+
+/* book pages */
+ul li.leaf {
```
Add ul li.expanded

ericras replied:
This wipes out list-style for all drupal lists but this pull request only fixes the list style for .block-menu-block

This should be moved to the other file and scoped to .block-menu-block

ericras commented on css/menu-block.css:
```diff
@@ -3,25 +3,28 @@
-.block-menu-block ul:first-child > li > ul > li > a {background-position: 1em center; background-repeat: no-repeat;}
-.block-menu-block li.collapsed > a {background-image: url("../../../../../misc/menu-collapsed.png");}
-.block-menu-block li.expanded > a {background-image: url("../../../../../misc/menu-expanded.png");}
+.block-menu-block ul:first-child > li > a {display: block; padding-left: 2.532em; background-color: #eae9e6;}
```
Move this below the next rule.  THe padding-left is getting overridden

ericras commented on css/menu-block.css:
```diff
@@ -3,25 +3,28 @@
 
-.block-menu-block a {color: #333;}
-.block-menu-block a:hover {color: #d00000;}
-.block-menu-block a.active {background-color: #F2E8B1 !important;}
```
I think it would be nice to have some sort of visual indicator on .active

skoolbus39 replied:
I agree and will explore options for the active state.

ericras commented on css/menu-block.css:
```diff
@@ -3,25 +3,28 @@
+.block-menu-block a:hover {cursor: pointer; color: #d00000;}
+.block-menu-block a.active {color: #6a0000;}
+.block-menu-block h3 {background-color: #474746; color: #fefdfa; font-size: .802rem; text-transform: uppercase; margin: 0; line-height: 1.333}
+.block-menu-block h3 a {display: block; padding-top: 1.333em; padding-bottom: 1.069em; padding-left: 1em; border: none; color: #fefdfa;}
```
Move padding to the h3 for the use case where the header isn't linked (an option in Menu Block)

skoolbus39 committed:
Move padding to h3 for instances where heading does not contain a link. https://github.com/unlcms/unl_fourone/pull/7/files#r60489617

skoolbus39 committed:
Move rule to avoid padding-left being overridden

Move rule further down to avoid padding-left being overridden. https://github.com/unlcms/unl_fourone/pull/7/files#r60489229

skoolbus39 committed:
Add ul li.expanded

https://github.com/unlcms/unl_fourone/pull/7/files#r60487850

skoolbus39 committed:
Move list-style to menu-block file

Move list-style to menu-block file to avoid Drupal list styling conflicts. https://github.com/unlcms/unl_fourone/pull/7/files#r60490256

skoolbus39 committed:
Change h3 a.active color

Dark red text on dark gray background doesn't work—using white instead.

skoolbus39 committed:
Padding/spacing adjustments menu block items

skoolbus39 committed:
Heading padding adjustments

ericras commented on css/menu-block.css:
```diff
@@ -3,25 +3,36 @@
+.block-menu-block li.collapsed > a:before {content: '\25B6';}
+.block-menu-block li.expanded > a:before {content: '\25BC';}
+.block-menu-block ul > li > ul > li > ul li.leaf > a:before {content: '\2022';}
+.block-menu-block ul li.leaf,
```
looks like we need "ul li.collapsed" here as well

skoolbus39 committed:
Remove list style type/image for ul li.collapsed

Addresses Eric's comment: https://github.com/unlcms/unl_fourone/pull/7#discussion_r60587696

skoolbus39 committed:
Add hover transitions

skoolbus39 committed:
Add better visual indicator for active state

Fixes https://github.com/unlcms/unl_fourone/pull/7#discussion_r60489477

skoolbus39 committed:
Remove duplicate style declarations

font-size already included in the declaration that immediately precedes it https://github.com/skoolbus39/unl_fourone/blob/d2243b7cc658d872e783faa17ae690e42114dee9/css/menu-block.css#L31

skoolbus39 committed:
Layout updates

* Adjust padding and positioning of list, anchor, and pseudo elements
* Adjust opacity of collapsed arrows, expanded arrows, and list bullets
